src/components/navigation/login/ProviderBlock.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="custom-provider-container">
+    <h3>{{providerName}}</h3>
+    <v-divider>
+    </v-divider>
+    <div class="custom-login-btn-container">
+      <v-btn v-for="entry of entries" @click="authenticate(entry.endpoint)" :key="entry.key"
+             class="login-btn" color="primary">
+        {{entry.key}}
```

Comment:
This value should be converted from slugified camelCase string to normal user viewable string. Meaning

wargaming => Wargaming
battleNet => Battle Net
etc....

So Upper letter in slugified string means space and first letter of id should always be capitalized with filter.

Reply:
Or run these trough translator...

Reply:
i'll resolve this as part of my next PR. I want to get Nath work in so I can get my PRs in.
Is it ok if we don't run through the translator as it is a company name it won't need translating, just formatting.
Thoughts?

Reply:
That is true. Rather not use translator. Ill resolve this, but this need to be fixed before release.